### ISSUE TITLE
[feature request] Auto Check for updates, and Auto re-load the page

### ISSUE THREAD

**Issue by toofast101** [jerrod-lankford/google-voice-desktop-app#99]
Hi, thanks for a great product, it's made Google Voice so much more useful to me.

It would be awesome if there was an option to Auto Check for updates, perhaps once a day, or whenever the software is loaded.

Also an option to Auto Reload the page whenever the window is opened. I get the plain white screen a lot, so wouldn't mind the extra time-overhead of fully reloading the page everytime it's opened.

**Comment by jerrod-lankford** [comment]
Yes auto update is high on the list, thanks for putting it in writing. I'll try to get around to working on it.
